pysatSpaceWeather/tests/test_data/3-day-geomag-forecast.txt

```diff
@@ -0,0 +1,25 @@
+:Product: Geomagnetic Forecast
```

Comment:
Is test_data used in addition to download tests or on its own? For this and the other sources.

Reply:
It's used in addition, you'll find the new test class for this later in your review.